packages/synthetics-sdk-broken-links/src/broken_links.ts

```diff
@@ -73,7 +73,8 @@ const synthetics_sdk_broken_links_package = require('../package.json');
 instantiateMetadata(synthetics_sdk_broken_links_package);
 
 export async function runBrokenLinks(
-  inputOptions: BrokenLinkCheckerOptions
+  inputOptions: BrokenLinkCheckerOptions,
+  total_timeout_millis = 250000
```

Comment:
Should total_timeout_millis be a member of BrokenLinkCheckerOptions?

Reply:
I don't think so... To me I wouldn't want to put a time limit on checking multiple links, especially if I'm not sure how many links I have (e.g. someone writes a link limit of 1000 to make sure all their links are checked).

Given that customers can set individual link timeouts I think this is probably sufficient. What do you think?